waspc/cli/src/Wasp/Cli/Interactive.hs

```diff
@@ -37,22 +39,35 @@ import qualified Wasp.Util.Terminal as Term
   We want to avoid this so users can type the name of the option when answering
   without having to type the quotes as well.
 
-  We introduced the Option class to get different "show" behavior for Strings and other
-  types. If we are using something other then String, an instance of Option needs to be defined,
+  We introduced the IsOption class to get different "show" behavior for Strings and other
+  types. If we are using something other then String, an instance of IsOption needs to be defined,
   but for Strings it just returns the String itself.
 -}
-class Option o where
+class IsOption o where
   showOption :: o -> String
   showOptionDescription :: o -> Maybe String
 
-instance Option [Char] where
+instance IsOption [Char] where
   showOption = id
   showOptionDescription = const Nothing
 
+data Option o = Option
```

Comment:
To learn about the motivation for this change, check comment https://github.com/wasp-lang/wasp/pull/1601/files#r1426920624 .